parser/parser.y

```diff
@@ -3653,6 +3657,20 @@ FunctionCallNonKeyword:
 	{
 		$$ = &ast.FuncCallExpr{FnName: model.NewCIStr($1), Args: $3.([]ast.ExprNode)}
 	}
+|	"JSON_EXTRACT" '(' Expression ',' Expression ')'
```

Comment:
It seems not right here
[JSON_EXTRACT](https://dev.mysql.com/doc/refman/5.7/en/json-search-functions.html#function_json-extract) can receive more than 2 arguments.